src/unit_tests/syscheckd/whodata/test_win_whodata.c

```diff
@@ -6342,11 +6352,47 @@ void test_run_whodata_scan_no_auto_audit_policies(void **state) {
     expect_value(__wrap_wm_exec, add_path, NULL);
     will_return(__wrap_wm_exec, 1);
     will_return(__wrap_wm_exec, 0);
+    expect_string(__wrap__merror, formatted_msg, "(6955): Auditpol command failed, attempt number: 1");
```

Comment:
This blocks could be compacted in the `expect_wm_exec` function, and maybe with a loop to call them.

Reply:
Addressed with https://github.com/wazuh/wazuh/pull/21455/commits/27c2e8f3c801ad04126261eb11f62862b9623a07